homeassistant/components/tts/__init__.py

```diff
@@ -379,6 +378,28 @@ class TextToSpeechEntity(RestoreEntity):
     _attr_should_poll = False
     __last_tts_loaded: str | None = None
 
+    _attr_default_language: str
+    _attr_supported_languages: list[str]
+    _attr_supported_options: list[str] | None = None
+    _attr_default_options: Mapping[str, Any] | None = None
+
+    @classmethod
+    def __init_subclass__(cls) -> None:
+        if not (
+            hasattr(cls, "_attr_default_language")
+            or type(cls.default_language) is property  # type: ignore[comparison-overlap]
```

Comment:
This is an not a correct way to check if the method has been overridden. Instead, check `cls.default_language is TextToSpeechEntity`